### DISCUSSION TITLE
Minio lifecycle bucket configuration is not working, can someone please check, the prefix is currently

### DISCUSSION THREAD

**Discussion by psrikar-uptycs** [minio/minio#17716] [Q&A] [answered]
## NOTE
If this case is urgent, please subscribe to [Subnet](https://min.io/pricing) so that our 24/7 support team may help you faster.

## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## Your Environment

* Version used (`minio --version`):
* Server setup and configuration:
* Operating System and version (`uname -a`):

**Comment by psrikar-uptycs** [comment]


**Reply by klauspost** [reply]
How have you determined it isn't working? Could you post your process and output from commands?

**Comment by psrikar-uptycs** [comment]


**Reply by klauspost** [reply, marked as answer]
Isn't that just prefixes? If you have versioning enabled prefixes will remain, if you are only creating delete markers.

Use `mc ls --versions ALIAS/bucket/path` to see what is actually there.

**Reply by psrikar-uptycs** [reply]
So the thing is we haven’t used mc in deploying minio, Followed the below link to install it as docker container
https://github.com/sakkiii/minio-docker-letsencrypt-deploy/blob/main/nginx/conf.d/minio.conf.template

**Comment by vadmeste** [comment]
> "Prefix": " "

this indicates there is a space in the prefix, could you double check if it is the case or not ?

**Comment by psrikar-uptycs** [comment]
Hey sorry, there is no space in prefix, some issue while copying

**Comment by psrikar-uptycs** [comment]
Issue got resolved, closing this conversation
